gapic-generator/templates/default/service/rest/client/method/def/_response_server_streaming.erb

```diff
@@ -0,0 +1,11 @@
+<%- assert_locals method -%>
+
+Gapic::Rest::ServerStream.new(
```

Comment:
Please prefix all fully-qualified types with `::`. i.e. `::Gapic::Rest::ServerStream`. This is to guard against the corner case of some other generated class using the name `Gapic` and messing with constant lookup. Same with all other template code below.